### ISSUE TITLE
Default ProgressBar shows up although renderLoader is customized

### ISSUE THREAD

**Issue by aspnetde** [react-pdf-viewer/react-pdf-viewer#421]
Hi there!

I've set up a custom loading indicator according to the docs:

```jsx
<Viewer
  ...
  renderLoader={_ => (
    <div ...>
      <LoadingIndicator />
    </div>
  )}
/>
```

However, I noticed that at some (seemingly random) points the default progress bar shows up. See the video attached (my loading indicator is that pulsing thing that sits vertically and horizontally centered, the default progress bar shows up on top).

https://user-images.githubusercontent.com/1322584/104343553-5b964a80-54fc-11eb-945a-2664611acf66.mov

Is there anything I may have misconfigured?

---

Update: In order to reproduce it, I kept reloading the page during the recording.

**Comment by SvetlankaKuzyaka** [comment]
Hi! The same issue - see [the video](https://vimeo.com/650197014/8553f1331d) Viewer component:
`<Viewer
  ...
  renderLoader={() => <Spinner size="xl" color="grey.400" />}
/>`

**Comment by phuoc-ng** [comment]
@aspnetde @SvetlankaKuzyaka @okilled
It is fixed in the [v3.1.0](https://react-pdf-viewer.dev/updates/). Please note that only the default `Spinner` at the loading document phrase is replaced. 
The option doesn't replace the `Spinner` in other places.
